Bottom: 30e615e6035ff19fc1524cea7d6d7f01a308b9f7
Top:    101e2e26d1b3058a941705c1dd819f8884758963
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:21 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
